src/screens/Home/HomeScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import {
+  View,
+  Text,
+  ScrollView,
+} from 'react-native'
+import styles from 'src/styles/globalStyles'
+
+const HomeScreen: React.FC = () => {
```

Comment:
I think we can remove part 'Screen' from 'HomeScreen', because we have this component in 'screens' folder already. But I'm not sure.

Reply:
Good catch!